core/providers/contractEvents/reducer.ts

```diff
@@ -14,7 +14,10 @@ interface RemoveContractEvent {
 
 type Action = AddContractEvent | RemoveContractEvent;
 
-export function contractEventReducer(state: ContractEvents, action: Action): ContractEvents {
+export function contractEventReducer(
```

Comment:
It looks like there is a lot of formatting. Does your IDE have a different config for max character length, etc?

Reply:
it was just `deno fmt`, should I revert this change?

Reply:
reverted the change, if needed or if u want can create a separate PR with `deno fmt`